cypress/integration/seededFlows/adminFlows/users/userIndexView.spec.js

```diff
@@ -1,4 +1,18 @@
+function openUserActionsDropdown() {
+  // This helper function allows us to use `pipe` to retry commands in case the test runner clicks before the JS has run
+  const click = (el) => el.click();
```

Comment:
Since I needed to access this within this helper function, I added this helper here and on [line 14](https://github.com/forem/forem/pull/17965/files#diff-dc32870eda99bf9b77089993f832ab6dcb8473a657bb2b8ddfc4e216b912a963R14) as well (by adding it to line 14, other areas of the test outside of the user actions dropdown that also need the helper can still access it).

Reply:
I wonder if at some point we want to have a "pipeHelpers" file or something, so we can import & use this in all the various tests that need it 🤔 

It feels a bit "extra" as it's such a small function but we do find ourselves having to do this a lot! Maybe we can think about it next time we need it 🙂

Reply:
That's a great idea! I believe we use it in a handful of specs, so that would be a solid refactor. I'll take note of this so that we can circle back around at some point. 😊